packages/adapter-cloudflare-workers/package.json

```diff
@@ -33,18 +33,20 @@
 		"index.d.ts"
 	],
 	"scripts": {
+		"build": "esbuild src/worker.js --bundle --outfile=files/worker.js --external:SERVER --external:MANIFEST --format=esm",
 		"lint": "prettier --check .",
 		"format": "pnpm lint --write",
-		"check": "tsc --skipLibCheck"
+		"check": "tsc --skipLibCheck",
+		"prepublishOnly": "pnpm build"
 	},
 	"dependencies": {
 		"@cloudflare/workers-types": "^4.20250312.0",
-		"esbuild": "^0.24.0"
+		"worktop": "0.8.0-next.18"
 	},
 	"devDependencies": {
-		"@cloudflare/kv-asset-handler": "^0.4.0",
 		"@sveltejs/kit": "workspace:^",
 		"@types/node": "^18.19.48",
+		"esbuild": "^0.24.0",
```

Comment:
why is this pinned to a prerelease and whats the benefit over the probably much leaner esbuild?